src/client/Client.js

```diff
@@ -459,15 +459,17 @@ export default class Client {
             callSettings.customData = null;
         }
         if (callSettings.extraHeaders === undefined) {
-            callSettings.extraHeaders = null;
+            callSettings.extraHeaders = {};
         }
         if (callSettings.setupCallKit === undefined || callSettings.setupCallKit === null) {
             callSettings.setupCallKit = false;
         }
```

Comment:
Please wrap it with Platform.OS === 'ios'

Reply:
done